### DISCUSSION TITLE
Can I open an program with a program that I added to Gamehub?

### DISCUSSION THREAD

**Discussion by ghost** [tkashkin/GameHub#470] [Q&A] [answered]
So what I mean is,for example,if you have a Text File and wanna open it with an Text Editor,then I could just right click that file in a File Manager and select the Text Editor of my choice to open it with that,but what if I wanna do the same thing with the Text Viewer that is added to Gamehub?
I know it doesn't make much sense to add a Text Editor to GameHub,but you get my point.

**Comment by tkashkin** [comment, marked as answer]
Not currently.

You can try to create a `.desktop` file in `~/.local/share/applications` that launches a specific game through GameHub.

You can launch a game with `com.github.tkashkin.gamehub --run <gameid>`,  but currently there's no option to override command line arguments of a game when launching it (should not be too hard to implement).

**Reply by ghost** [reply]
I did a [feature request](https://github.com/tkashkin/GameHub/issues/479),cause that feature would be cool and making shortcuts manually isn't as fast.
